doc/api/modules.md

```diff
@@ -841,11 +841,19 @@ added: v9.3.0
 A list of  the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third-party module or not.
 
+Note that `module` in this context isn't the same object that's provided
```

Comment:
Should this note be moved to below the example for accessing `builtinModules`? Its placement here make `it` in `To access it` (the next sentence) seem like it refers to the module wrapper which is kind of the opposite of the sentences intention, I think?

Reply:
I was actually referring to the `module` mentioned in the headline of this section just above. But I can see that this can be a bit confusing. Not sure if moving it below the code example helps much unless I also change the code example to assign the required module to a constant like I did in the alternative code example in the PR description. What do you think?

Reply:
This is why I like to refer to this module with an uppercase M as it makes it much easier to distinguish between the two